internal/backend/sessions/sessionworkflows/workflows.go

```diff
@@ -217,9 +217,15 @@ func (ws *workflows) sessionWorkflow(wctx workflow.Context, params *sessionWorkf
 
 		startTime = data.Session.CreatedAt()
 		l.Info("session workflow: started", fields...)
+
 		prints, err = runWorkflow(wctx, l, ws, data, params.Debug)
+
 		duration = time.Since(startTime)
-		sessionDurationHistogram.Record(ctx, duration.Milliseconds())
+		sessionStatus := "ok"
+		if err != nil {
+			sessionStatus = "error"
+		}
+		sessionDurationHistogram.Record(ctx, duration.Milliseconds(), telemetry.WithLabels("status", sessionStatus))
```

Comment:
also add replay

Reply:
why replay?
we count the total session run from created to final state
replay is just an internal implementation detail

Reply:
@itayd 
moved to https://linear.app/autokitteh/issue/ENG-1462/metrics-3. Resolving here